services/api/src/routes/keys.ts

```diff
@@ -82,7 +81,7 @@ const keysRoute = async (
     );
   }
 
-  res.send(result);
```

Comment:
This errors here when tried to access with a key thats not in Lagoon (and this trips up anything using the ssh agent)
```
[2024-06-30 23:15:51] [user_auth]: Legacy authentication granted for 'ssh' from 'unknown': {"user":{"id":"ssh","role":"admin","aud":"api.dev","iat":1719789351,"permissions":{}},"headers":{"user-agent":"Wget/1.21.2","content-type":"application/json","content-length":"69","host":"api:3000","ipAddress":"::ffff:172.22.0.23"},"payload":null}
[2024-06-30 23:15:51] [debug]: Accessing keys with fingerprint: SHA256:Kc6CB00BxZHGzQC4UwXSOFsdM3c27HPe4bmpp8hiF0Q
[2024-06-30 23:15:51] [debug]: Unknown fingerprint: SHA256:Kc6CB00BxZHGzQC4UwXSOFsdM3c27HPe4bmpp8hiF0Q
[2024-06-30 23:15:51] [error]: uncaughtException: Cannot read properties of undefined (reading 'key')
TypeError: Cannot read properties of undefined (reading 'key')
    at /app/services/api/dist/routes/keys.js:100:23
    at Generator.next (<anonymous>)
    at fulfilled (/app/services/api/dist/routes/keys.js:28:58)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
[2024-06-30 23:15:51] [error]: uncaughtException: Cannot read properties of undefined (reading 'key')
TypeError: Cannot read properties of undefined (reading 'key')
    at /app/services/api/dist/routes/keys.js:100:23
    at Generator.next (<anonymous>)
    at fulfilled (/app/services/api/dist/routes/keys.js:28:58)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
[winston] Unknown logger level: error
```

Reply:
ugh merges

Reply:
theres also an issue with existing keys being shown as unknown (and proceeding)
```
[2024-06-30 23:28:55] [debug]: Accessing keys with fingerprint: SHA256:inQGcrMz0Bp0fTovkhOQgH70z8sMU8jjZbrHSw2MPN4
[2024-06-30 23:28:55] [debug]: Unknown fingerprint: SHA256:inQGcrMz0Bp0fTovkhOQgH70z8sMU8jjZbrHSw2MPN4
```

Reply:
You may need to pull your changes, the `res.send(result);` was removed already

Reply:
Oh wait :see_no_evil: